app/src/main/res/values/strings_help.xml

```diff
@@ -5,69 +5,54 @@
     <string name="help_title">Help</string>
     <string name="help_navigation_title">Navigation</string>
     <string name="help_navigation_text">
-        From the home screen, swipe down to display the favorites panel and swipe up to hide it.
+        Swipe down from the home screen to display the favorites panel, and upwards to hide it.
         \n
-        \nIf you swipe up when the favorites panel is hidden, it will display the complete list of applications.
-        Swipe down from the top of this list to come back to the home screen.
+        \nSwipe up when the favorites panel is hidden, to show all apps in a list.
+        Swipe down from the top of it to return to the home screen.
         \n
-        \nPress the button below the favorites panel to display the menu of the launcher.
+        \nThe button below the favorites panel opens the launcher menu.
         \n
-        \nIf you hold your finger a few moments on an application or a folder, you will be offered to access its settings.
+        \nLong-hold an app or folder to access its settings.
     </string>
     <string name="help_notification_title">Notification</string>
     <string name="help_notification_text">
-        The notification allows to display a popup with favorites above other applications when you leave the home screen.
-        In most cases, it should work directly and without issues.
+        Shows a popup of favorites above other apps when you leave the home screen.
         \n
-        \nHowever, in some special cases it may bring back to the home screen instead of being displayed over other applications.
-        \nIf you are facing this issue, there are two things you can try:
-        \n- Press the \"clear all\" button in Recent Apps
-        \n- Reboot your smartphone
-        \nAfter one of these actions, the problem should not appear again.
-        \nIf it persists, I am sorry but it seems this will not work for you :-/
+        \If instead it brings you back to the home screen:

```

Comment:
How much of a problem is this? Can it be removed?

Reply:
Ok for me to remove it.
In the firsts versions where the notification was added, it was sometimes needed to do one or the other fix to get it work correctly. From release to release this has been progressively corrected and I think now it is quite stable.

*Side note: all the modifications above are OK for me, it is indeed a lot simpler like this!*

Reply:
I tried to do without the word "panel", which isn't used anywhere else.